rmgpy/molecule/group.py

```diff
@@ -858,16 +858,17 @@ class Group(Graph):
     Corresponding alias methods have also been provided.
     """
 
-    def __init__(self, atoms=None, multiplicity=None):
+    def __init__(self, atoms=None, props=None, multiplicity=None):
         Graph.__init__(self, atoms)
-        self.multiplicity = multiplicity if multiplicity else []
+        self.props = props or {}
+        self.multiplicity = multiplicity or []
```

Comment:
It mirrors the `props` attribute of `Molecule()`, which I just found was added by Nick to mirror the `props` attribute of `Species()`. It's just a placeholder for arbitrary properties/flags that would be suitable for a dictionary format.